jenkins/fpm.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+version=2.46.1
+
+fpm -v ${version} \
+    --iteration 1.el7 \
+    --epoch 1 \
+    --vendor EDINA \
+    --provides jenkins \
+    -a noarch \
+    -m john.pinto@ed.ac.uk \
```

Comment:
Its actually quite annoying specifying the Java dependency as we're using the Oracle JDK, which has the version as part of the name.